wire-compiler/src/main/java/com/squareup/wire/schema/Target.kt

```diff
@@ -64,6 +64,33 @@ sealed class Target {
   interface SchemaHandler {
     fun handle(type: Type)
     fun handle(service: Service)
```

Comment:
remove the above methods?

Reply:
Actually I kind of need them for Java/Kotlin/Custom target otherwise the default implementation of `handle(protofile)` is doomed